tests/performance/performance.cpp

```diff
@@ -233,8 +260,59 @@ typedef class _ebpf_program_test_state
     _program_info_provider program_info_provider;
 } ebpf_program_test_state_t;
 
+typedef class _ebpf_map_test_state
+{
+  public:
+    _ebpf_map_test_state(ebpf_map_type_t type)
+    {
+        ebpf_utf8_string_t name{(uint8_t*)"test", 4};
+        ebpf_map_definition_in_memory_t definition{
+            sizeof(ebpf_map_definition_in_memory_t), type, sizeof(uint32_t), sizeof(uint64_t), 1};
+
+        REQUIRE(
+            ebpf_map_create(&name, &definition, reinterpret_cast<uintptr_t>(ebpf_handle_invalid), &map) ==
+            EBPF_SUCCESS);
+    }
+    ~_ebpf_map_test_state() { ebpf_object_release_reference((ebpf_object_t*)map); }
+
+    void
+    test_find_read()
+    {
+        uint32_t key = 0;
+        uint64_t* value = nullptr;
+        ebpf_epoch_enter();
+        ebpf_map_find_entry(map, 0, (uint8_t*)&key, 0, (uint8_t*)&value, EBPF_MAP_FLAG_HELPER);
+        ebpf_epoch_exit();
+    }
+
+    void
+    test_find_write()
+    {
+        uint32_t key = 0;
+        uint64_t* value = nullptr;
+        ebpf_epoch_enter();
+        ebpf_map_find_entry(map, 0, (uint8_t*)&key, 0, (uint8_t*)&value, EBPF_MAP_FLAG_HELPER);
```

Comment:
this looks identical to line 284.   I suspect having two identical functions with different names (read vs write) is not what you intended.

Reply:
Good catch.